UnitTests/BraintreePayPalTests/BTPayPalReturnURL_Tests.swift

```diff
@@ -0,0 +1,40 @@
+import XCTest
+@testable import BraintreePayPal
+
+final class BTPayPalReturnURL_Tests: XCTestCase {
```

Comment:
Most of this test class is not new, was just renamed. The `testInitWithSchemeURL` are the only new tests since we are passing both scheme based URLs and Universal Link URLs into the `init` now.